src/include/common/statement.h

```diff
@@ -68,22 +69,22 @@ class Statement {
 
  private:
   // logical name of statement
-  std::string statement_name;
+  std::string statement_name_;
 
   // query string
-  std::string query_string;
+  std::string query_string_;
 
   // first token in query
-  std::string query_type;
+  std::string query_type_;
```

Comment:
Yes, this is the long-term goal. As I had mentioned to you that day, using type::StatementType for situations like these requires us to support multi-query statements first. Once I get that patch in, all these things can be made cleaner.